css/main.css

```diff
@@ -1,11 +1,35 @@
+:root {
+  --color-bg: #fff;
+  --color-text: #fff;
+  --color-text-secondary: #000;
+  --color-what-is-title-bg: #1d1283;
+  --color-why-description-text: #3d3d3d;
+  --color-social-media-links-bg: #fff;
+  --color-links: #1d1283;
+  --color-events: #e30162;
+}
+
+.dark-theme {
```

Comment:
Could we have a separate css file, where we will have `lightTheme` and `darkTheme` classes? And let's import those classes wherever needed.